pipeline/filters/log_to_metrics.md

```diff
@@ -39,12 +40,12 @@ The following example takes records from two dummy inputs and counts all message
 
 [INPUT]
     Name               dummy
-    Dummy              {"message":"dummy", "kubernetes":{"namespace_name": "default", "docker_id": "abc123", "pod_name": "pod1", "container_name": "mycontainer", "pod_id": "def456"}, "duration": 20, "color": "red", "shape": "circle"}
+    Dummy              {"message":"dummy", "kubernetes":{"namespace_name": "default", "docker_id": "abc123", "pod_name": "pod1", "container_name": "mycontainer", "pod_id": "def456", "labels":{"app": "app1"}}, "duration": 20, "color": "red", "shape": "circle"}
     Tag                dummy.log
 
 [INPUT]
     Name               dummy
-    Dummy              {"message":"hello", "kubernetes":{"namespace_name": "default", "docker_id": "abc123", "pod_name": "pod1", "container_name": "mycontainer", "pod_id": "def456"}, "duration": 60, "color": "blue", "shape": "square"}
+    Dummy              {"message":"hello", "kubernetes":{"namespace_name": "default", "docker_id": "abc123", "pod_name": "pod1", "container_name": "mycontainer", "pod_id": "def456" "labels":{"app": "app1"}}, "duration": 60, "color": "blue", "shape": "square"}
```

Comment:
It seems that a missing "," is between  "def456" and "labels". Without it, the change makes invalid JSON parameter.

Reply:
Fixed in df180040a4996e9aacc7be53c04de91d056c17fc.